tests/pub-external-links/EPUB/package.opf

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Publication Resources</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-03-06</dc:date>
+    <dc:description>
+        Reading Systems SHOULD open links that resolve outside the EPUB Publication in a new browser instance
+        to ensure that the security and privacy controls of the browser are available to users.
+    </dc:description>
+    <dc:identifier id="pub-id">pub-external-links</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>External Links should be opened a new browser instance</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-epub-rs-external-links</meta>
```

Comment:
Per [the spec](https://www.w3.org/TR/epub-rs-33/#sec-epub-rs-external-links) this feature is a SHOULD and not a MUST, ie, the relevant meta element must be added.

```suggestion
    <meta property="belongs-to-collection">should</meta>
    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-epub-rs-external-links</meta>
```

Reply:
thanks for pointing out. will fix.